.github/workflows/docker-build.yaml

```diff
@@ -72,3 +72,5 @@ jobs:
       run: |
         docker run -d -p 5000:5000 --name registry registry:2
         make bundle-build bundle-push catalog-build
```

Comment:
We also need to update add the `catalog-deps-build` command here, right?

Reply:
Also, to the `docker-push` action

Reply:
No there is no need for that, That is required only for the openshift CI

Reply:
Ok, I see we have that in 4.16 and main branch. So, shall we remove that from there as well?

Reply:
It's okay we can keep it there, It is not required at least on the old branch.

Reply:
I have backported that commit as well.

Reply:
Cool 😅